public/pages/VisualCreatePolicy/components/UIActions/ForceMergeUIAction.tsx

```diff
@@ -30,23 +30,24 @@ export default class ForceMergeUIAction implements UIAction<ForceMergeAction> {
 
   clone = (action: ForceMergeAction) => new ForceMergeUIAction(action, this.id);
 
+  isValid = () => {
+    const segments = this.action.force_merge.max_num_segments;
+    return !!segments && segments > 0;
+  };
+
   render = (action: UIAction<ForceMergeAction>, onChangeAction: (action: UIAction<ForceMergeAction>) => void) => {
+    const segments = action.action.force_merge.max_num_segments;
     return (
       <>
-        <EuiFormCustomLabel title="Max num segments" helpText="The number of segments to merge to." />
-        <EuiFormRow isInvalid={false} error={null}>
+        <EuiFormCustomLabel title="Max num segments" helpText="The number of segments to merge to." isInvalid={!this.isValid()} />
+        <EuiFormRow isInvalid={!this.isValid()} error={null}>
```

Comment:
May want to give the error message according to isValid() logic, like "Please provide a positive number."

Reply:
Yeah, just probably won't get to it in this PR. Will see if we can get that in before the release, if not can be an enhancement for the next one.